net/datacenter-node-install/scripts/disable-nouveau.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cat <<EOF > /etc/modprobe.d/blacklist-nouveau.conf
+blacklist nouveau
+options nouveau modeset=0
```

Comment:
Setting options on a blacklisted module? :thinking:

Reply:
well...it works the way it is